pkg/satellite/constants.go

```diff
@@ -0,0 +1,17 @@
+package satellite
+
+const (
```

Comment:
I would add the consts near the respective places rather than a consts file. And if it's only used once or twice, it might not help with readability.